# request, nothing to pass in here

---

# response, true if successful, false if not

bool reset
